common-tasks/dispvm.md

```diff
@@ -20,7 +20,30 @@ Once a DispVM has been created it will appear in Qubes VM Manager with the name
 
 See [this article](https://blog.invisiblethings.org/2010/06/01/disposable-vms.html) for more on why one would want to use a Disposable VM.
 
-Disposable VMs and Networking
+Disposable VMs and Networking (R4.0 and later)
+-----------------------------
+
+
+R4.0 introduces the concept of multiple disposable VM templates (R3.2 was limited to one).
+This allows for the creation of multiple differently configured disposable VMs that can be accessed from 
+the Applications menu. Even more types of DispVMs can be created on-the-fly on a per AppVM basis.
+As you can see, this is a very flexible and powerful system for managing your Disposable VMs.
+
+NetVM and firewall rules for Disposable VMs can be set as they can for a normal VM. 
+By default a DispVM will inherit the NetVM and firewall settings of the DispVM Template from which it is built. 
+Thus if an AppVM uses sys-net as its NetVM, but the default system DispVM uses sys-whonix,
+any DispVM launched from this AppVM will have sys-whonix as its NetVM.
```

Comment:
Aren't these two sentences contradictory?

Reply:
Also, I think we need to first explain that an AppVM can be a DVM Template.

Reply:
Not contradictory- it's saying that under 4.0, launching a DispVM from an AppVM will result in it using the DispVM's network/firewall settings (which default to the DVM template from which it was built). Compare & contrast with the same section a little further down under 3.2 which has different behaviour. It's definitely hard to follow, but I'm not sure how else to phrase it...

Reply:
I could add "in other words" and include that same sentence from my comment, if it helps understand what's going on?

Reply:
We need to clearly explain the difference between the "based on" and "launched from" relationships, and we need provide examples for the other main cases.

In your example, I launch a DispVM from an AppVM using `sys-net`, but the DispVM doesn't use `sys-net`. Instead, it uses `sys-whonix`, because it's based on my default DVM Template, which uses `sys-whonix`. I can imagine a user having several questions about this:
 - Will the same pattern of behavior occur if I launch a DispVM from `anon-whonix` while my default DVM Template uses `sys-net`? That would be incredibly dangerous.
 - Why was my DispVM based on the default DVM Template rather than the AppVM from which I launched it?
 - How do I base the DispVM on the AppVM instead?

Some of these questions are answered below, but in a way that at least partially contradicts what you wrote above (e.g., because it depends on the settings explained below).